Library/Homebrew/cask/quarantine.rb

```diff
@@ -174,5 +175,18 @@ def self.propagate(from: nil, to: nil)
 
       raise CaskQuarantinePropagationError.new(to, quarantiner.stderr)
     end
+
+    def self.copy_xattrs(from, to)
+      odebug "Copying xattrs from #{from} to #{to}"
+
+      copier = system_command!(swift,
+                               args: [
+                                 *swift_target_args,
+                                 COPY_XATTRS_SCRIPT,
+                                 from,
+                                 to,
+                               ])
+      copier.success?
```

Comment:
This is useless, since `system_command!` will fail on error.

Reply:
Fixed.